src/Uno.Extensions.Http/IAuthenticationToken.cs

```diff
@@ -0,0 +1,8 @@
+﻿using System.Threading.Tasks;
+
+namespace Uno.Extensions.Http;
+
+public  interface IAuthenticationToken
```

Comment:
Should we rename to IAuthenticationTokenProvider ?

Reply:
Yep will do this